pwa/static/themes/rotterdam.css

```diff
@@ -134,6 +134,7 @@
   --web-app-footer-logo-container-height: 50px;
   --web-app-footer-organization-logo-width: 230px;
   --web-app-footer-organization-logo-src: url("images/LogoRotterdam.svg");
+  --web-app-footer-organization-logo-wit-src: url("images/LogoRotterdamWit.svg");
```

Comment:
Nice addition. However, when adding a new design token to a theme it's _very_ important that we add them to the other themes as well.

In this case: we need to add it to the base theme ".web-app-theme" _and_ the Utrecht theme ".utrecht-theme".

The effect of not adding the new token to the other themes:


![image](https://github.com/OpenCatalogi/web-app/assets/66552932/f8698514-c6f1-40bb-80bd-15897c8be970)

Reply:
Sorry this pr should be closed, the improved footer is in this [PR](https://github.com/OpenCatalogi/web-app/pull/277)